SpectraLogic.SpectraStorageBrokerClient/SpectraStorageBrokerClient.cs

```diff
@@ -86,20 +86,20 @@ public IDevice CreateDevice(CreateDeviceRequest request)
             });
         }
 
-        public IJob Delete(DeleteRequest request)
+        public void DeleteCluster()
         {
-            return ExceptionDecorator.Run(() =>
+            ExceptionDecorator.Run(() =>
             {
-                return new DeleteResponseParser().Parse(_network.Invoke(request));
+                var request = new DeleteClusterRequest();
+                return new DeleteClusterResponseParser().Parse(_network.Invoke(request));
             });
         }
 
-        public void DeleteCluster()
+        public IJob DeleteFiles(DeleteFilesRequest request)
```

Comment:
I don't know that deleting files will return a job.  It's an interesting idea though that we may want to consider.  Are deletes asynchronous is really the question we need to answer.

Reply:
Just a placeholder